lib/compat/wordpress-6.0/theme.json

```diff
@@ -185,6 +185,85 @@
 			],
 			"text": true
 		},
+		"layout": {
```

Comment:
For ease of review, I've kept these changes in the `6.0` file for `theme.json`. If the PR looks viable, we can move this to a `6.1` file before merging.